ocaml-lsp-server/src/document.mli

```diff
@@ -97,7 +97,7 @@ val close : t -> unit Fiber.t
     counterparts for the URI [uri].
 
     For instance, the counterparts of the file [/file.ml] are [/file.mli]. *)
-val get_impl_intf_counterparts : Uri.t -> Uri.t list
+val get_impl_intf_counterparts : Merlin.t option -> Uri.t -> Uri.t list
```

Comment:
I kept this to accept `Merlin.t option` as I understand it still could work for docs outside of the current LSP workspace (e.g. do jump to definition to some opam installed lib and then still can use this switch impl/intf command to switch between .ml/.mli).